internal/ark/backups_svc.go

```diff
@@ -105,6 +105,23 @@ func (s *BackupsService) List() ([]*api.Backup, error) {
 	return backups, nil
 }
 
+// List returns Backup instances for a given cluster
+func (s *BackupsService) ListForACluster(clusterID uint) ([]*api.Backup, error) {
+	backups := make([]*api.Backup, 0)
+
+	items, err := s.repository.FindByClusterID(clusterID)
+	if err != nil {
+		return backups, err
+	}
+
```

Comment:
**Recommended**: could reinitialize slice here with `backups = make([]int, 0, len(items))` to optimize allocations while also keeping the `append()` style filling.

Reply:
fixed